storage/src/test/resources/log4j2.yaml

```diff
@@ -28,8 +28,8 @@ Configuration:
         pattern: "${logPattern}"
     RollingFile:
       - name: FileAppender
-        fileName: storage.log
-        filePattern: "storage-%d{yyyy-MM-dd}.log"
+        fileName: /tmp/kafka-storage-test/storage.log
```

Comment:
It appears the log files are intended for debugging test failures. Therefore, placing these files in the 'build' directory, which is cleaned by the cleanTest task, would be a more appropriate location.